deploy/deploy-operator-cs/MyStack.cs

```diff
@@ -9,281 +9,299 @@ class MyStack : Stack
-        {
-            Spec = new Kubernetes.Types.Inputs.Apps.V1.DeploymentSpecArgs
+            });
+            var operatorDeployment = new Kubernetes.Apps.V1.Deployment($"pulumi-kubernetes-operator-{ns}", new DeploymentArgs
             {
-                Replicas = 1,
-                Selector = new LabelSelectorArgs
+                Metadata = new ObjectMetaArgs{Namespace = ns},
+                Spec = new Kubernetes.Types.Inputs.Apps.V1.DeploymentSpecArgs
                 {
-                    MatchLabels = 
+                    Replicas = 1,
+                    Selector = new LabelSelectorArgs
                     {
-                        { "name", "pulumi-kubernetes-operator" },
-                    },
-                },
-                Template = new PodTemplateSpecArgs
-                {
-                    Metadata = new ObjectMetaArgs
-                    {
-                        Labels = 
+                        MatchLabels = 
                         {
                             { "name", "pulumi-kubernetes-operator" },
                         },
                     },
-                    Spec = new PodSpecArgs
+                    Template = new PodTemplateSpecArgs
                     {
-                        ServiceAccountName = operatorServiceAccount.Metadata.Apply(md => md.Name),
-                        Containers = 
+                        Metadata = new ObjectMetaArgs
                         {
-                            new ContainerArgs
+                            Labels = 
                             {
-                                Name = "pulumi-kubernetes-operator",
-                                Image = "pulumi/pulumi-kubernetes-operator:v1.8.0",
-                                Command = 
-                                {
-                                    "pulumi-kubernetes-operator",
-                                },
-                                Args = 
-                                {
-                                    "--zap-level=debug",
-                                    "--zap-time-encoding=iso8601",
-                                },
-                                ImagePullPolicy = "Always",
-                                Env = 
+                                { "name", "pulumi-kubernetes-operator" },
+                            },
+                        },
+                        Spec = new PodSpecArgs
+                        {
+                            ServiceAccountName = operatorServiceAccount.Metadata.Apply(md => md.Name),
+                            Containers = 
+                            {
+                                new ContainerArgs
                                 {
-                                    new EnvVarArgs
+                                    Name = "pulumi-kubernetes-operator",
+                                    Image = "pulumi/pulumi-kubernetes-operator:v1.8.0",
```

Comment:
Guessing all of the image references will need to be updated.

Reply:
Yesssss hey how did that work before? (_someone or something_ updated them to 1.8.0 ...)

Reply:
Oh, looks like I may have missed doing that on the release. I think Vivek had manually bumped those references previously. https://github.com/pulumi/pulumi-kubernetes-operator/pull/315

Reply:
I logged https://github.com/pulumi/pulumi-kubernetes-operator/issues/334 to automate it (but yeah, manual bump this time)